index.php

```diff
@@ -111,9 +111,12 @@
 
 		<!-- Customer Review Form -->
 		<div class="row">
-			<div class="col-6">
+			<div class="col-6" id="IndexReviewForm">
 				<?php include 'forms/review.form.php'; ?><br />
```

Comment:
@billythakidd04 same thing with this. I wouldn't have touched this with this ticket. My thinking is that I pulled in some changes at some point and did something with the incoming changes.

Reply:
Interesting. Check your git log and see if theres a commit that may need to be reverted.

Reply:
You can see the changes in a specific commit by doing `git show <commit_hash>`